File: unitazIO/unitaz.py
from .pooping_exceptions import UnitazIsFull


class Unitaz:
    pooped_times = 0

    @classmethod
    def iteratively_pooped(cls) -> None:
        if cls.pooped_times == 10:
            raise UnitazIsFull('Пришло время смыть за собой!')

        cls.pooped_times += 1

File: unitazTS/unitaz.ts
export class Unitaz {
    private poopedTimes: number = 0;

    protected iterativelyPooped(): any {
        if (this.poopedTimes === 10) {
            throw new Error('Пришло время смыть за собой! Вы покакали уже 10 раз подряд!');
        }

        this.poopedTimes += 1;
    }
}
